src/App.jsx

```diff
@@ -14,33 +16,77 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
+  const reset = () => {
+    setSortField('reset');
+    setReversed(false);
+  };
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  goodsFromServer.sort((good1, good2) => {
+    switch (sortField) {
+      case 'length':
+        return good1.length - good2.length;
+      case 'alphabet':
+        return good1.localeCompare(good2);
+      default:
+        return 0;
+    }
+  });
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (reversed) {
+    goodsFromServer.reverse();
```

Comment:
Reversing `goodsFromServer` directly mutates the exported array. Consider creating a copy of the array before reversing, similar to the sorting suggestion.